### PR TITLE
Layout container

### PR DESCRIPTION
**Pull request by s-laugh**
### Description

fixing #320 

List of proposed changes:

- Moved LayoutContainer implementation to the Layout component for each page
- Removed the use of it from individual pages along with redundant divs

### Additional Notes

None of the content of the pages was edited at all, just the line indenting.

### CODE REVIEW

**Review by CurtisUnderwood — CHANGES_REQUESTED** [review]
image

Some issues with the splash page. Looks like it's still taking the footer somehow and the background image isn't stretching the full page

**Comment by s-laugh** [comment]
> Some issues with the splash page. Looks like it's still taking the footer somehow and the background image isn't stretching the full page

Funny enough, the footer has been there for a while... both issues fixed now.

File: components/organisms/Layout.js [hunk 1]
```diff
@@ -78,7 +79,11 @@ export default function Layout(props) {
       )}
 
       <main>
-        <div>{props.children}</div>
+        {props.fullscreen ? (
+          props.children
+        ) : (
+          <LayoutContainer>{props.children}</LayoutContainer>
+        )}
```
Comment (DaceyTom2) [inline components/organisms/Layout.js:86]: I think my review got deleted but layout and layoutContainer should be applied to _app so they don't have to be added to pages

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: components/organisms/Layout.js (hunk 1 above)
Comment (s-laugh) [reply components/organisms/Layout.js]: Wasn't quite in the scope of what I was aiming to do, but I've added that change as well.

**Review by DaceyTom2 — APPROVED** [review]
Looks great!

**Review by CurtisUnderwood — APPROVED** [review]
Nice work!

**Pull request merged**